src/components/Search/SearchPageHeader.tsx

```diff
@@ -314,7 +314,14 @@ function SearchPageHeader({queryJSON, hash, onSelectDeleteOption, setOfflineModa
             <Button
                 text={translate('search.filtersHeader')}
                 icon={Expensicons.Filters}
-                onPress={() => Navigation.navigate(ROUTES.SEARCH_ADVANCED_FILTERS)}
+                onPress={() => {
+                    const filters = SearchUtils.getFilters(queryJSON);
+                    const form = SearchUtils.getFiltersForm(filters);
+                    form.type = queryJSON.type;
+                    form.status = queryJSON.status;
```

Comment:
you created `. getFiltersForm()` method to use in this place specifically - so it should do all the work needed for creating form data.
Because of that it should set `form.type` and `status` inside itself, so that when you call it, all the data is prepared.

You can either pass it `type` and `status` as separate arguments, or just pass whole `queryJSON` as the 2nd param - I think both are ok.